R/bambu_utilityFunctions.R

```diff
@@ -22,19 +25,18 @@ setIsoreParameters <- function(isoreParameters){
         min.readFractionByGene = 0.05,
         min.sampleNumber = 1,
         min.exonDistance = 35,
-        min.exonOverlap = 10, #
+        min.exonOverlap = 10, 
         min.primarySecondaryDist = 5,
         min.primarySecondaryDistStartEnd1 = 5, # for creating new annotations
         min.primarySecondaryDistStartEnd2 = 5, # for read assignment
-        min.exonOverlap = 10,
         min.txScore.multiExon = 0,
         min.txScore.singleExon = 1,
         fitReadClassModel = TRUE,
         defaultModels = defaultModels,
         returnModel = FALSE,
         baselineFDR = 0.1,
         min.readFractionByEqClass = 0,
-        prefix = "") 
+        prefix = "Bambu") 
```

Comment:
I think prefix suits being a list argument since its a very niche case that it will actually be changed, so I would rather not have argument bloat. Thoughts?